pages/linux/lshw.md

```diff
@@ -13,8 +13,8 @@
 
 - List all disks and storage controllers in tabular format:
 
-`sudo lshw -class disk -class storage -short`
+`sudo lshw {{[-c|-class]}} disk -class storage -short`
 
-- Save all network interfaces to an HTML file:
+- Save all network interfaces to an HTML/XML/JSON file:
 
-`sudo lshw -class network -html > {{interfaces.html}}`
+`sudo lshw {{[-c|-class]}} network -{{html|xml|json}} > interfaces{{.html|.xml|.json}}`
```

Comment:
```suggestion
`sudo lshw {{[-c|-class]}} network -{{html|xml|json}} > interfaces.{{html|xml|json}}`
```

Another possibility. Either is fine with me.

Reply:
I like my example better because it's clearer that we're talking about an extention.